WcaOnRails/app/inputs/money_amount_input.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+class MoneyAmountInput < SimpleForm::Inputs::Base
+  def input(wrapper_options)
+    merged_input_options = merge_wrapper_options(input_html_options, wrapper_options)
+
+    actual_field = @builder.hidden_field(attribute_name)
+    input_id = attribute_name.to_s + "_input_field"
+    script = template.content_tag(:script, type: 'text/javascript') do
+      raw("wca.setupCurrencyMask('##{input_id}', '##{@builder.object_name}_#{attribute_name}', '#{currency}');")
```

Comment:
I didn't want to mess with how simple_form generate/interact with names, so instead I did it a way similar to what we do for the autocomplete fields: set a custom css class, and enable the mask for them on load.